packages/apt/test.sh

```diff
@@ -185,6 +187,26 @@ if [ "${can_upgrade}" = "yes" ]; then
   run_test
 fi
 
+echo "::group::Downgrade"
+
+if [ "${can_downgrade}" = "yes" ]; then
+  pgroonga_current_version=$(apt-cache search --full \^${pgroonga_package}\$ | \
```

Comment:
Can we use other name than `current`?
It seems that you use `current` for the latest released version but I feel that `current` means that the testing version.

Reply:
I changed variable name to `pgroonga_latest_release_version` from `pgroonga_current_version`.

Reply:
`released` will be better than `release`.

Reply:
I used `released` instead of `release`.